components/email-input.jsx

```diff
@@ -39,14 +39,27 @@ var EmailInput = React.createClass({
     });
     this.props.onChange(this.props.name, this);
   },
-  render: function() {
+  renderHint: function() {
     var hintClassIconName = "fa fa-question-circle hint";
     var hintClassName = "hint-msg small";
+    var info = this.props.info;
     if (this.state.showHint) {
       hintClassIconName += " on";
     } else {
       hintClassName += " hidden";
     }
+    if (info) {
```

Comment:
Email component can be used in multiple forms, so making the hint generic and optional is probably ideal.